Entities/Characters/Builder/BuilderLogic.as

```diff
@@ -326,7 +327,8 @@ void Pickaxe(CBlob@ this)
 	@bestinfo = hit_p.bestinfo;
 	bestDistance = hit_p.bestDistance;
 
-	bool noBuildZone = map.getSectorAtPosition(tilepos, "no build") !is null;
+	Tile tile = map.getTile(tilepos);
+	bool noBuildZone = inNoBuildZone(map, tilepos, tile.type);
```

Comment:
What's this doing? Looks unnecessary.

Reply:
Do you mean the `noBuildZone` variable? It is used a couple lines below, old code.

Reply:
Oh right. I didn't see that.